build/params_mainnet.go

```diff
@@ -126,3 +127,7 @@ const BootstrapPeerThreshold = 4
 
 // we skip checks on message validity in this block to sidestep the zero-bls signature
 var WhitelistedBlock = MustParseCid("bafy2bzaceapyg2uyzk7vueh3xccxkuwbz3nxewjyguoxvhx77malc2lzn2ybi")
+
+// CBDeliveryDelay is the delay before deliver in the synchronous consistent broadcast.
+// This determines the wait time for the detection of potential equivocations.
+var CBDeliveryDelay = 2 * time.Second
```

Comment:
Do we not have to add this variable to the other params files too? Are you able to (for example) `make calibnet` successfully?

Also why is it a var and not a const here?

Reply:
Good catch, it should be a `const` and it should be an all testnet builds for them to compile. Fixed. Thanks!